x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation.cy.ts

```diff
@@ -126,7 +115,8 @@ describe.skip('Timelines', { tags: ['@ess', '@serverless'] }, (): void => {
       .should('have.text', getTimeline().notes);
   });
 
-  it('shows the different timeline states', () => {
+  it('should show the different timeline states', () => {
+    login();
```

Comment:
line 121 we have a `createTimeline` method but I feel like we might have another method used in other places already?

Also looking at its implementation, you can remove the `should('be.visible')` as Cypress already checks for that before clicking

Reply:
`createTimeline` is used throughout this file. I'm not aware of another way to create one.

Reply:
Ah, it seems like `createTimeline` did basically the same as `openTimelineUsingToggle`. So I replaced the former with the latter everywhere (https://github.com/elastic/kibana/pull/173413/commits/729e5ceaf20034988ea6e4ba0c50bb130e59e715).